lib/librato/rails/subscribers/job.rb

```diff
@@ -0,0 +1,15 @@
+module Librato
+  module Rails
+    module Subscribers
+      hooks = %w{enqueue_at enqueue perform_start perform}
+
+      hooks.each do |hook|
+        ActiveSupport::Notifications.subscribe "#{hook}.active_job" do |*args|
+          collector.group 'rails.job' do |c|
+            c.increment hook
```

Comment:
do any of these provide any interesting information other than that they ran? timing info, etc?